harper-dictionary-parsing/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "harper-dictionary-parsing"
+version = "0.12.0"
+edition = "2021"
+description = "The language checker for developers."
+license = "Apache-2.0"
+repository = "https://github.com/elijah-potter/harper"
+
+[dependencies]
+blanket = "0.4.0"
+hashbrown = { version = "0.15.0", features = ["serde"] }
```

Comment:
`cargo +nightly udeps` suggests that `blanket`, `itertools`, `ordered-float`, `paste`, `unicode-blocks`, and `unicode-width` aren't needed.